docs/guides/building-forms.md

```diff
@@ -393,8 +387,8 @@ That's about it. After reading this guide, here's what you'll learn:
 - The basic HTML `form` element
 - Understanding forms with React.js
 - Validating forms data with and without JavaScript
-- Using Next.js Sereverless Functions to handle `req` and `res` from the client and server
+- Using Next.js to handle `req` and `res` from the client and server
```

Comment:
Could we make this a little clearer and say using API Routes?

Reply:
Done.